server/src/main/java/org/elasticsearch/threadpool/ThreadPool.java

```diff
@@ -934,7 +933,7 @@ public static class Info implements Writeable, ToXContentFragment {
         private final int min;
         private final int max;
         private final TimeValue keepAlive;
-        private final SizeValue queueSize;
+        private final Long queueSize;
```

Comment:
The only possible observable change is that the `Info#toString()` now always shows the bare integer rather than something like `10k` for larger numbers. I'm not sure this is actually visible to end-users anywhere and honestly IMO that's an improvement.